aws/kops/main.tf

```diff
@@ -69,86 +69,85 @@ locals {
-      overwrite   = "true"
-      description = "Kops state store (S3 bucket) URL"
-    },
-    {
-      name        = "${format(var.chamber_parameter_name, local.chamber_service, "kops_state_store_region")}"
-      value       = "${module.kops_state_backend.bucket_region}"
-      type        = "String"
-      overwrite   = "true"
-      description = "Kops state store (S3 bucket) region"
-    },
-    {
-      name        = "${format(var.chamber_parameter_name, local.chamber_service, "kops_ssh_public_key_path")}"
-      value       = "${module.ssh_key_pair.public_key_filename}"
-      type        = "String"
-      overwrite   = "true"
-      description = "Kops SSH public key filename path"
-    },
-    {
-      name        = "${format(var.chamber_parameter_name, local.chamber_service, "kops_ssh_private_key_path")}"
-      value       = "${module.ssh_key_pair.private_key_filename}"
-      type        = "String"
-      overwrite   = "true"
-      description = "Kops SSH private key filename path"
-    },
-    {
-      name        = "${format(var.chamber_parameter_name, local.chamber_service, "kops_dns_zone")}"
-      value       = "${module.kops_state_backend.zone_name}"
-      type        = "String"
-      overwrite   = "true"
-      description = "Kops cluster name"
-    },
-    {
-      name        = "${format(var.chamber_parameter_name, local.chamber_service, "kops_network_cidr")}"
-      value       = "${var.network_cidr}"
-      type        = "String"
-      overwrite   = "true"
-      description = "CIDR block of the kops virtual network"
-    },
-    {
-      name        = "${format(var.chamber_parameter_name, local.chamber_service, "kops_private_subnets")}"
-      value       = "${local.private_subnets}"
-      type        = "String"
-      overwrite   = "true"
-      description = "Kops private subnet CIDRs"
-    },
-    {
-      name        = "${format(var.chamber_parameter_name, local.chamber_service, "kops_utility_subnets")}"
-      value       = "${local.utility_subnets}"
-      type        = "String"
-      overwrite   = "true"
-      description = "Kops utility subnet CIDRs"
-    },
-    {
-      name        = "${format(var.chamber_parameter_name, local.chamber_service, "kops_availability_zones")}"
-      value       = "${join(",", local.availability_zones)}"
-      type        = "String"
-      overwrite   = "true"
-      description = "Kops availability zones in which cluster will be provisioned"
-    },
-  ]
+# These parameters correspond to the kops manifest template:
+# Read more: <https://github.com/cloudposse/geodesic/blob/master/rootfs/templates/kops/default.yaml>
+
+resource "aws_ssm_parameter" "kops_cluster_name" {
+  name        = "${format(var.chamber_parameter_name, local.chamber_service, "kops_cluster_name")}"
+  value       = "${module.kops_state_backend.zone_name}"
+  description = "Kops cluster name"
+  type        = "String"
+  overwrite   = "true"
+}
+
+resource "aws_ssm_parameter" "kops_state_store" {
```

Comment:
```suggestion
resource "aws_ssm_parameter" "kops_state_store_bucket_name" {
```

Reply:
`kops_state_store` is what `kops` calls it.   The objective is to use the same envs as `kops` expects to avoid any additional mapping.

https://github.com/kubernetes/kops/blob/master/docs/aws.md#prepare-local-environment